From foo@baz Mon Jan  5 15:04:44 PST 2009
Date: Mon, 05 Jan 2009 15:04:44 -0800
To: Greg KH <greg@kroah.com>
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Staging: benet: fix netif api breakage

The netif_rx_complete() and netif_rx_schedule() functions have changed,
so fix up the build breakage in the benet driver.

Cc: Subbu Seetharaman <subbus@serverengines.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/benet/be_int.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/staging/benet/be_int.c
+++ b/drivers/staging/benet/be_int.c
@@ -728,7 +728,7 @@ int be_poll(struct napi_struct *napi, in
 
 	/* All consumed */
 	if (work_done < budget) {
-		netif_rx_complete(netdev, napi);
+		netif_rx_complete(napi);
 		/* enable intr */
 		be_notify_cmpl(pnob, work_done, pnob->rx_cq_id, 1);
 	} else {
@@ -763,7 +763,7 @@ static inline u32 process_events(struct 
 		rid = AMAP_GET_BITS_PTR(EQ_ENTRY, ResourceID, eqp);
 		if (rid == pnob->rx_cq_id) {
 			adapter->be_stat.bes_rx_events++;
-			netif_rx_schedule(netdev, &pnob->napi);
+			netif_rx_schedule(&pnob->napi);
 		} else if (rid == pnob->tx_cq_id) {
 			process_nic_tx_completions(pnob);
 		} else if (rid == pnob->mcc_cq_id) {
